common/app/views/emailSubscriptionNonsuccessResult.scala.html

```diff
@@ -1,23 +1,6 @@
-@import model.{InvalidEmail, OtherError, NonsuccessResult}
-
+@import model.NonsuccessResult
 @(page: model.Page, result: NonsuccessResult)(implicit request: RequestHeader, context: model.ApplicationContext)
 
-@emailEmbed(page){
-    <div class="email-sub email-sub--article">
-        <div class="email-sub__message">
-        @result match {
-            case InvalidEmail => {
-                @fragments.inlineSvg("cross", "icon")
-                <h2 class="email-sub__message__headline">Invalid email address</h2>
-                <p class="email-sub__message__description">Sorry, we couldn't verify that email address. Please try again or <a target="_blank" href="https://www.theguardian.com/info/tech-feedback">contact user help</a>.</p>
-            }
-
-            case OtherError => {
-                @fragments.inlineSvg("cross", "icon")
-                <h2 class="email-sub__message__headline">Subscription error</h2>
-                <p class="email-sub__message__description">Sorry, there was a problem with your subscription request. Please try again or <a target="_blank" href="https://www.theguardian.com/info/tech-feedback">contact user help</a>.</p>
-            }
-        }
-        </div>
-    </div>
+@emailEmbed(page) {
+    @fragments.email.signup.result.emailNonsuccess(result)
```

Comment:
Does this still need to be put in a separate file? (or is it for consistency or something else I missed?)

Reply:
It's for consistency more than anything else in this PR. I would like to have the error states and non-valid email address states be displayed in the new design rather than as text on a blank background, so this would help for any of that future work as well.